app/views/vocab_sheets/_vocab_sheet_item.html.haml

```diff
@@ -1,12 +1,12 @@
-%li.vocab_sheet_item.vocab_sheet_page_item.shadow.rounded{:style => "width:#{vocab_sheet_item_width}px;height:#{vocab_sheet_item_height}"}
-  = hidden_field_tag "item[#{vocab_sheet_item.id}][id]", vocab_sheet_item.id, :class => 'item_id'
-  = link_to sign_url(vocab_sheet_item.sign_id), {:class => 'drawing vocab_sheet_drawing', :style => "line-height:#{vocab_sheet_image_height}px;height:#{vocab_sheet_image_height}px"} do
```

Comment:
These inline styles with interpolation a pretty leftfield.
Definitely room to work on simplifying views and styles.
Will add to my Friday radar!

Reply:
Sweet... I like that the code base is a million times better than when we picked it up years ago..
the Haml needs some clean up


the JS is the worst though.